components/TextFields/README.md

```diff
@@ -55,13 +59,13 @@ pod install
 
 ### Overview
 
-Text Fields provides both a single-line version based on UITextField and a multi-line version backed by UITextView as well as objects that customize the text fields' behavior and appearance called 'Text Input Controllers'.
+Text Fields provides both a single-line version based on `UITextField` and a multi-line version backed by `UITextView` as well as objects that customize the text fields' behavior and appearance called 'Text Input Controllers'.
 
-The actual components (MDCTextField & MDCMultilineTextField) are 'dumb': they do not have styles, animations, or advanced features. They are designed to be controlled from the outside, via very liberal public API, with a text input controller.
+The actual components (`MDCTextField` & `MDCMultilineTextField`) are 'dumb': they do not have styles, animations, or advanced features. They are designed to be controlled from the outside, via very liberal public API, with a text input controller.
 
-One text input controller is included (MDCTextInputControllerDefault) manipulates the exposed elements of the text field to make placeholders float.
+Most text input controllers included are based off of `MDCTextInputControllerDefault` which manipulates the exposed elements of the text field to make placeholders float.
```

Comment:
Grammar nit: "are based on" instead of "are based off of"

Reply:
Done.